src/util.js

```diff
@@ -57,8 +57,32 @@
     return false;
   }
 
+
+  /*
+   * Get iterator of object which returns a { done: false, value: expression }
+   *   if it has not reached the end of the object, or otherwise it returns
+   *   { done: true, value: undefined }.
+   * @param {!Object} object: The object to iterate over.
+   * @param {Function} callback: A function to be called by using key and value
```

Comment:
callback: Defines the value attribute for the iterator. The function takes two arguments (key, value), and should return one of the following: key, value, or an array containing one or more of (key, value). Generally, key is used for the keys() method, value is used for the values() method, and [key, value] for the entries() method.